docs/publications.rst

```diff
@@ -0,0 +1,97 @@
+:orphan:
+
+.. _publications:
+
+############
+Publications
+############
+
+Below is a list of recent publications that use the BEAST:
+
+* Multiwavelength Characterization of the High Mass X-ray Binary Population of M31
+
+  Lazzarini et al., 2020
+
+  https://ui.adsabs.harvard.edu/abs/2020arXiv201007273L/abstract
```

Comment:
This comment applies to all the references.  

It would be cleaner to have the URL as a link to the title.  There is an RST format for this - checkout the naming convention page for `dust_extinction` for this syntax or google RST links. 

You could also consider having the link text include the authors/year as well and have everything on one line.  More compact and possible easier to scan/read?

Reply:
Fixed!

Reply:
But better than nothing!  ;-)

We can always revise the ordering/make sections in a later PR.